src/http_rest_api_handler.cpp

```diff
@@ -526,7 +526,12 @@ Status HttpRestApiHandler::processMetrics(const HttpRequestComponents& request_c
         return StatusCode::INTERNAL_ERROR;  // TODO: Return proper code when metric endpoint is disabled (missing module).
     }
 
-    auto servableManagerModule = dynamic_cast<const ServableManagerModule*>(module);
+    auto manager_module = this->ovmsServer.getModule(SERVABLE_MANAGER_MODULE_NAME);
```

Comment:
Consider changing the name of this variable to match camel case convention - `managerModule`